src/components/ResourceList/components/Item/Item.tsx

```diff
@@ -159,14 +168,17 @@ class Item extends React.Component<CombinedProps, State> {
           testID="LargerSelectionArea"
         >
           <div onClick={stopPropagation} className={styles.CheckboxWrapper}>
-            <div onChange={this.handleLargerSelectionArea}>
+            <div>
               <Checkbox
                 testID="Checkbox"
                 id={this.checkboxId}
                 label={label}
                 labelHidden
                 checked={selected}
                 disabled={loading}
+                onChange={() => {
+                  this.handleLargerSelectionArea();
+                }}
```

Comment:
onChange={this.handleLargerSelectionArea};

Reply:
I believe I tried that and hit a TypeScript wall. Will sort it out.

Reply:
Ah so the reason I did this was because `handleLargerSelectionArea` expects an `event` that it can stop propagation on but `Checkbox`'s `onChange` just passes a boolean. To make this work I made `handleLargerSelectionArea`'s `event` argument optional (https://github.com/Shopify/polaris-react/pull/1706/files#diff-425411853250df4be3bd738e2b2bab02R342) and called it with no arguments from here.